README.md

```diff
@@ -0,0 +1,175 @@
+For Node.js Developers
+======================
+
+The JSON Schema Test Suite is meant to be a language agnostic test suite for testing JSON Schema validation libraries.
+It is generally added to projects as a git submodule. However, to simplify things for Node.js developers, the test suite has also
+been made available as an npm package.
+
+    npm install json-schema-test-suite
```

Comment:
It looks like below in the package.json that the package name is changing (to `json-schema-test-suite-npm` -- does that affect this line?

Reply:
@Julian Should we try and publish using the `json-schema-test-suite` name or change to `json-schema-test-suite-npm` ?

Reply:
Isn't the `-npm` suffix redundant given that it will be accessible on npm? I also rememberr npm was encouraging developers to stop using `node` prefixes/suffixes on packages.

Reply:
@LucianBuzzo I mean if we had access to it yeah I'd stick with the old one, but I don't have access to that name -- do you?

Reply:
@Julian @jviotti I'm going to reach out to @subfuzion (Tony Pujals) who is reported as the owner of the package on npm and see if he can add me as a collaborator